wrappers/python/nntile/model/llama_config.py

```diff
@@ -0,0 +1,67 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/nntile/model/llama_config.py
+# LLaMa model config
+#
+# @version 1.0.0
+
+from typing import Dict
+
+
+class LlamaConfigNNTile(Dict):
```

Comment:
Why not use `dataclass` decorator? @daskol could you please advice if there is an easy `dataclass`-like `dict` decorator? To support `dict`-like access while using simplicity of `dataclass`?

Reply:
Use `dict` in favor of `dataclass` does not make a lot of sense if all attributes are known a priori.

```python
from dataclasses import asdict, dataclass

@dataclass
class Config: ...

config = Config(...)
value = asdict(config)['key']['subkey']['subsubkey']
```